plutus-playground/plutus-playground-client/src/MainFrame.purs

```diff
@@ -196,54 +199,68 @@ eval (LoadScript key next) = do
       void $ withEditor $ Editor.setValue contents (Just 1)
       assign _evaluationResult NotAsked
       assign _compilationResult NotAsked
-      assign _actions []
```

Comment:
do you not need to assign `Nothing` to signatures?

Reply:
No, not here. Imagine you:

* Load a file, compile it, and create a simulation.
* Make some changes to the file.
* Regret those changes, and click the load button again to revert.

In that case, I think you'd want the simulation to still be there. I think we should only reset the signature if compilation yields one that's valid _and_ different. That's handled elsewhere.